src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -108,6 +106,16 @@ class ENAButton: DynamicTypeButton {
 	private func applyHighlight() {
 		highlightView.isHidden = !isHighlighted
 	}
+
+	private func setHeight() {
+		let heightConstraint = heightAnchor.constraint(greaterThanOrEqualToConstant: 50)
```

Comment:
Do we have some option don't use constraints inside the ENAButton? Maybe `contentInsets` or similar ? It's hard to predict how it would work in `UIStackView`.

Reply:
This is the only option to make the button title resize on system font changes that I could find that works. And when I implemented this originally, on Invite Friends screen, I lost hours trying different options. 

When I originally implemented this, I also had it on Reset screen, in a stack view, and it worked. It's still the case, as far as I could see.